website/docs/usage/examples.md

```diff
@@ -161,8 +161,7 @@ https://github.com/explosion/spaCy/tree/master/examples/training/train_textcat.p
 ### Visualizing spaCy vectors in TensorBoard {#tensorboard}
 
 This script lets you load any spaCy model containing word vectors into
-[TensorBoard](https://projector.tensorflow.org/) to create an
-[embedding visualization](https://www.tensorflow.org/versions/r1.1/get_started/embedding_viz).
+[TensorBoard](https://projector.tensorflow.org/) to create an embedding visualization.
```

Comment:
I'm also not sure what the original link contained, but this seems to be a similar tutorial from Tensorflow, covering some of the concepts used in spaCy's example script:
```suggestion
[TensorBoard](https://projector.tensorflow.org/) to create an
[embedding visualization](https://github.com/tensorflow/tensorboard/blob/master/docs/tensorboard_projector_plugin.ipynb).
```

Reply:
Great suggestion. Thanks for digging this up! I've committed your suggestion.